app/views/claims/_ineligibility_reason_ineligible_qts_award_year.html.erb

```diff
@@ -0,0 +1,4 @@
+<p class="govuk-body">
+  You can only get this payment if you completed your initial teacher training
+  on or after September 1st 2013.
```

Comment:
This looks like an inconsistent date format, I think it should be `1 September 2013`

Reply:
Good spot.